app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java

```diff
@@ -90,10 +111,50 @@ public void onDrawerOpened(View drawerView) {
         };
         mDrawerLayout.setDrawerListener(mDrawerToggle);
 
+        String homeChapterId = getSharedPreferences("gdg", MODE_PRIVATE).getString(Const.SETTINGS_HOME_GDG, null);
```

Comment:
Changing the old code is hard but I think we should stop using Strings like "gdg" for the future.